pyaedt/desktop.py

```diff
@@ -153,28 +153,34 @@ def release_desktop(close_projects=True, close_desktop=True):
         ``True`` when successful, ``False`` when failed.
 
     """
-    if _com == "pythonnet":
-        Module = sys.modules['__main__']
+    Module = sys.modules['__main__']
+    if sys.modules['__main__'].interpreter == "ironpython":
         desktop = Module.oDesktop
         scopeID = desktop.ScopeID
         i = 0
         if close_projects:
             proj_list = desktop.GetProjectList()
             for prj in proj_list:
                 desktop.CloseProject(prj)
-
-        while i <= scopeID:
-            Module.COMUtil.ReleaseCOMObjectScope(Module.COMUtil.PInvokeProxyAPI, i)
-            i += 1
-        try:
-            del Module.oDesktop
-            return True
-        except:
-            Module.oMessenger.add_info_message("Attributes not present")
-            return False
-
-    elif _com == "pythonnet_v3":
-        Module = sys.modules['__main__']
+        if "COMUtil" in dir(Module):
+            while i <= scopeID:
+                Module.COMUtil.ReleaseCOMObjectScope(Module.COMUtil.PInvokeProxyAPI, i)
+                i += 1
+            try:
+                del Module.oDesktop
+                return True
+            except:
+                Module.oMessenger.add_info_message("Attributes not present")
+                return False
+        else:
+            pid = Module.oDesktop.GetProcessID()
+            try:
+                os.kill(pid, 9)
+                return True
+            except:
+                Module.oMessenger.add_error_message("something went wrong in Closing AEDT")
```

Comment:
Best to not let errors pass silently and to provide good logging.

```suggestion
            except Exception as e:
                Module.oMessenger.add_error_message("Failed to close AEDT:\n%s" % str(e))
```